src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx

```diff
@@ -49,7 +49,7 @@ If a company we're working with has its trademark policy publicly available or h
 
 You must use title case for our trademarked names including our company name plus our product/platform name and a few capability and integration names.
 
-This table is kept up to date and is the source of truth for capitalization. **Most capability and feature names are lowercase because New Relic is one platform.** However, some terms and capabilities are capitalized for trademark or pricing reasons. Note that the use of "capability" here is different than our use of that word in the [user management space](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions).
+This table is kept up to date and is the source of truth for capitalization. **Most capability and feature names are lowercase because New Relic is one platform.** However, some terms and capabilities are capitalized for trademark or pricing (for example, it's a separate SKU) reasons. Note that the use of "capability" here is different than our use of that word in the [user management space](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions).
```

Comment:
```suggestion
This table is kept up to date and is the source of truth for capitalization. **Most capability and feature names are lowercase because New Relic is one platform.** However, some terms and capabilities are capitalized for trademark or pricing reasons (for example, it's a separate SKU). Note that the use of "capability" here is different than our use of that word in the [user management space](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions).
```

Reply:
That looks good, Robert. Thanks for reviewing.